rust/src/filetracker.rs

```diff
@@ -15,17 +15,19 @@
  * 02110-1301, USA.
  */
 
+//! Gap handling and Chunk-based file transfer tracker module.
+//!
+//! GAP handling. If a data gap is encountered, the file is truncated
+//! and new data is no longer pushed down to the lower level APIs.
+//! The tracker does continue to follow the file.
+//!
+//! Tracks chunk based file transfers. Chunks may be transferred out
+//! of order, but cannot be transferred in parallel. So only one
+//! chunk at a time.
+
 /**
  *  \file
- *  \author Victor Julien <victor@inliniac.net>
- *
- * Tracks chunk based file transfers. Chunks may be transferred out
- * of order, but cannot be transferred in parallel. So only one
- * chunk at a time.
- *
- * GAP handling. If a data gap is encountered, the file is truncated
- * and new data is no longer pushed down to the lower level APIs.
- * The tracker does continue to follow the file.
+ *  \author Victor Julien <victor@inliniac.net> 
```

Comment:
where please?

Reply:
at the end of the line

Reply:
Another nit: from what I saw from this discussion, the idea was to merge both doc comments, keeping only one style.

So, instead of leaving ` \file` and `\author`, and the more c-like comment style, we'd do:
(see next comment)